dev/CommandBarFlyout/CommandBarFlyoutCommandBar.cpp

```diff
@@ -755,6 +763,27 @@ void CommandBarFlyoutCommandBar::EnsureFocusedPrimaryCommand()
     }
 }
 
+void CommandBarFlyoutCommandBar::MoreButtonClicked(const winrt::IInspectable& sender, const winrt::RoutedEventArgs& args)
```

Comment:
>MoreButtonClicked [](start = 33, length = 17)

Does this work with touch, where the Tapped event would get used?

Reply:
Touch works as expected.